proposal/chililn0525/README.rst

```diff
@@ -0,0 +1,116 @@
+=========
+autopitch
+=========
+
+Basic Information
+=================
+
+**Autopitch** is a tool to tune your pitch in real time.
+
+The GitHub repository : https://github.com/chilin0525/autopitch
+
+Problem to Solve
+================
+
+I had a terrible experience of singing test in music class when I was a senior high school student. The teacher asked me go back to the sit and spend more time practicing after I just sang first lyric. But the problem is that I could not understand how bad of my performance since I could not recognize the pitch. So I want to build a tool to tune pitch in real time. There exist a software call **autotune** also can tune your pitch, but it needs many experiences and tuned manually.  This is why the project named **autopitch** since we can use the tool to tune the pitch automatically.
+
+The steps to solve the problem:
+
+1. Receive and parse audio files, we should understand how computer store audio signal at first.
```

Comment:
If you need to read an audio _file_, how can the system be real-time?

Reply:
It makes sense to focus on the off-line analysis in the setting of a semester course.  I will suggest to tone down the (presumptively long-term) objective of making it real-time.